src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,344 @@
+
+####Everything Else
+
+Now that the study area is broken down logically into a hierarchy within the OpenHDS the demographic data has a geographical context. Every piece of information within the OpenHDS is tied in some way to the hierarchy, just as a population and its changes over time are tied to their geographical location. 
+
+The information in the OpenHDS comes in 3 main flavors. *Census* data: people, the places they live and the groups they belong to. *Relation* data: information about how people are related to one another, members of groups, or resident at a location. *Update* data: records of events that change the census data over time.
+
+#####A simplified overview of the OpenHDS entities and events.
+![Figure 2](figure-2.png)
+*figure 2*
+
+The **Census Entities** are Location, Individual, and SocialGroup.
+
++ **Locations** are particular points of interest within the study area. Locations are the leaf nodes on the hierarchy described above, there is nothing below them because they are most atomic geographical denomination. Examples of Locations could be buildings and houses. 
+ 
++ **Individuals** are the people within the area of study, i.e. the population. Since the OpenHDS is primarily focused on demographic information, many of the other entities and events within the system are dependent on individuals.
+
++ **SocialGroups** are the societal or cultural organizations that the people in the area of study are a part of. Examples of SocialGroups are church groups, families, book clubs, etc.
+
+The **Relation Entities** are Membership, Relationship, and Residency. 
+
++ **Memberships** are what tie an Individual and a SocialGroup together; a Membership is made for every SocialGroup a person a part of. An example of a Membership is being a member of a particular family. 
+
++ **Relationships** are what tie two Individuals together; a Relationship can be made for each way an individual is related to anyone else. An example of a Relationship is being the son of someone, being the pastor of someone, or being someone’s lawyer.
+
++ **Residencies** are what tie Locations and Individuals together; a Residency is made for each Location an Individual is a resident at. An example of a Residency is being a resident at your house (a Location) and perhaps your parent’s house (A different Location).
+
+Without explaining the Update Events yet, a picture is starting to form on how the Census and Relation Entities are oriented with respect to one another. To give things more context, however, it is necessary to explain the FieldWorker entity. The OpenHDS is a system that relies on data being collected in an area of study. To help keep track of who collected what and when, the FieldWorker is needed. All of the entities in *figure 2* are collected either directly or indirectly by a FieldWorker and thus all are dependent on the FieldWorker. Without Update Events, here is a look at how things are dependent on each other…

```

Comment:
I think FieldWorker deserves it's own **heading** and introduction, rather than being introduced in the middle of a paragraph.